tests/integration/submission/test_upload.py

```diff
@@ -409,9 +484,14 @@ def test_data_file_update_url_id_provided_different_file_already_indexed(
     assert_single_entity_from_response(resp)
 
 
-@pytest.mark.config_toggle(parameters={'ENFORCE_FILE_HASH_SIZE_UNIQUENESS': False})
+@pytest.mark.config_toggle(
+    parameters={
+        'ENFORCE_FILE_HASH_SIZE_UNIQUENESS': False
```

Comment:
FYI pytest-flask has a fixture for this purpose: 
http://pytest-flask.readthedocs.io/en/latest/features.html#config-application-config

Reply:
Yeah, but unfortunately we are not using the config and instead loading it in `app` fixture.